src/Administration/Resources/app/administration/src/module/sw-sales-channel/component/sw-sales-channel-detail-domains/sw-sales-channel-detail-domains.html.twig

```diff
@@ -68,6 +68,12 @@
             </template>
             {% endblock %}
 
+            <template
+                #column-measurementSystemId="{ item, isInlineEdit}"
+            >
+                {{ item.measurementSystem.translated.name }}
```

Comment:
```suggestion
                {{ item.measurementSystem.translated.name || item.measurementSystem.name  }}
```

Reply:
Shall we leave the word "System" right after the measurement name? @vienthuong @ralfhoefer 

```html
Metric system
```

Reply:
I think we don't need it as we declared the field as "Measurement system: Metric"